content/providers/01-ai-sdk-providers/170-baseten.mdx

```diff
@@ -0,0 +1,242 @@
+---
+title: Baseten
+description: Learn how to use Baseten models with the AI SDK.
+---
+
+# Baseten Provider
+
+[Baseten](https://baseten.ai/) is a platform for deploying and serving machine learning models through their [API](https://docs.baseten.co/overview).
```

Comment:
nit: this sentence reads a little weird ("platform for serving ... models through their API", the "their" is a bit ambiguous). Maybe split into a second sentence "serving machine learning models. They support this via their API", or reword.